uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/NotificationTray.kt

```diff
@@ -83,9 +105,9 @@ class NotificationTray @Inject constructor(
         return (id % Int.MAX_VALUE).toInt()
     }
 
-    private fun reshowAll() {
+    fun reshowAll() {
         for ((habit, data) in active.entries) {
-            taskRunner.execute(ShowNotificationTask(habit, data))
+            taskRunner.execute(ShowNotificationTask(habit, data, true))
```

Comment:
For boolean arguments, it's best to add the argument name (`shown = true`)

Reply:
Done.